tests/commands/install_uninstall_test.py

```diff
@@ -726,6 +726,51 @@ def test_commit_msg_legacy(commit_msg_repo, tempdir_factory, store):
     assert second_line.startswith('Must have "Signed off by:"...')
 
 
+def test_post_commit_integration(tempdir_factory, store):
+    path = git_dir(tempdir_factory)
+    config = [
+        {
+            'repo': 'local',
+            'hooks': [{
+                'id': 'post-commit',
+                'name': 'Post commit',
+                'entry': 'bash -c "touch post-commit.tmp"',
+                'language': 'system',
+                'always_run': True,
+                'verbose': True,
+                'stages': ['post-commit'],
+            }],
+        },
+        {'repo': 'meta', 'hooks': [{'id': 'identity'}]},
+    ]
+    write_config(path, config)
+    with cwd(path):
+        cmd_output('git', 'add', '.')
+        git_commit()
+
+        # assertion works
+        assert not os.path.exists('post-commit.tmp')
+
+        open('some_file', 'a').close()
+        cmd_output('git', 'add', '.')
+        git_commit()
+
+        # file is not created on every commit
+        assert not os.path.exists('post-commit.tmp')
+
+        install(C.CONFIG_FILE, store, hook_types=['post-commit'])
+
+        # file is not created by installation
+        assert not os.path.exists('post-commit.tmp')
```

Comment:
??? why assert this, I'd remove this

Reply:
The idea was to be assured that installation has no hidden call, but it's probably checked elsewhere. I'm removing it.